spec/features/locations/update_location_spec.rb

```diff
@@ -1,7 +1,8 @@
 describe "Update location", type: :feature do
-  let(:organisation) { create(:organisation, locations: [location]) }
-  let(:user) { create(:user, :confirm_all_memberships, organisations: [organisation]) }
   let(:location) { create(:location) }
+  let(:organisation) { location.organisation }
```

Comment:
looks good, but maybe add a request spec, so we know that a user can't set the location to an org they're not a member of